static/js/containers/LearnerSearchPage_test.js

```diff
@@ -230,7 +230,7 @@ describe('LearnerSearchPage', function () {
         query: 'xyz',
         type: 'phrase_prefix',
       });
-      assert.equal(window.location.toString(), "http://fake/?q=xyz");
+      assert.equal(window.location.toString(), "http://edx.org/?q=xyz");
```

Comment:
Why is this change necessary? If the problem is that `window.location` from one test is affecting this one, we should solve the problem in `babelhook.js` or `global_init.js` instead, where we handle `window.location` mocking